.github/workflows/test-phpunit.yml

```diff
@@ -169,6 +169,15 @@ jobs:
         env:
           COVERAGE_NAME: php-v${{ env.COVERAGE_PHP_VERSION }}-${{ matrix.db-platforms }}
 
+      - name: Cache PHPUnit coverage cache
+        if: matrix.php-versions == env.COVERAGE_PHP_VERSION
+        uses: actions/cache@v3
+        with:
+          path: build/.phpunit.cache/code-coverage
+          key: phpunit-code-coverage-${{ hashFiles('**/phpunit.*') }}
```

Comment:
Is the intent to hit the Config files and workflow definition itself? This seems like it could hit a lot more, like some of the output files themselves.

Reply:
Yeah, you're right! The intent was for the output but hashing them didn't cross my mind. 😅

Reply:
I decided to go with the first implementation because it _just_ works.

- Hashing the output files as filename will not work since those output files are not yet available here, so resulting in an empty string
- Uploading the results won't work because of the way download-artifact unzips the download

Reply:
If it works, I'm good with it until it doesn't 🤷‍♂️